components/mailbox/src/init.spec.js

```diff
@@ -8,7 +8,7 @@ describe("MailBox  component", () => {
     cy.visitMailbox();
   });
 
-  it("Shows Mailbox with demo id and threads", () => {
+  xit("Shows Mailbox with demo id and threads", () => {
```

Comment:
Should this still be `x`'d out?

Reply:
oh damn forgot to remove that, thanks! @ozsivanov